examples/stm32f103c8t6_blue_pill/tlc594x/main.cpp

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2019, Linas Nikiperavicius
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+// ----------------------------------------------------------------------------
+
+#include <modm/board.hpp>
+#include <modm/driver/pwm/tlc594x.hpp>
+
+using namespace Board;
+
+/*
+ * Turns on each TLC594* output in ascending order.
+ * Each output is active for 500 milliseconds.
+ */
+
+#define CHANNELS 24
+
+modm::TLC594X< CHANNELS, modm::platform::SpiMaster1, modm::platform::GpioA4, modm::platform::GpioA3 > tlc594x;
+
+int
+main()
+{
+	Board::initialize();
+
+	modm::platform::SpiMaster1::connect< modm::platform::GpioA5::Sck, modm::platform::GpioA7::Mosi > ();
```

Comment:
No need for `modm::platform` everywhere, the board.hpp file includes the namespace.

Reply:
Ok, removed them.